### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Anisha7** [MariomcgeeArt/checklist#1]
- Your tests should be able to run independent from your program. Try changing your tests such that it prints out output by testing different input, without requiring user input! The users should only be using your program, not testing it too. Your commented tests are a better approach than calling the select function, which would ask for user input.
- Please use profound language in your code and your print messages! I recommend changing your message on line 101. Expect that your code could be looked at by other engineers and/or potential recruiters.

**Comment by MariomcgeeArt** [comment]
Can I go over this with you?

**Comment by MariomcgeeArt** [comment]
Because I'm not quite sure what you mean

**Comment by Anisha7** [comment]
Yeah, of course! Come to my office hours (Thursdays 1:30-3:30) or any other TA's that works with your schedule.
